pkg/gits/github.go

```diff
@@ -791,25 +791,51 @@ func (p *GitHubProvider) ListOpenPullRequests(owner string, repo string) ([]*Git
 	return answer, nil
 }
 
+func extractRepositoryCommitAuthor(rc *github.RepositoryCommit) (gu *GitUser) {
+	commitAuthorPresent := rc.Commit.Author != nil
+	repositoryAuthorPresent := rc.Author != nil
+	repositoryAuthorOnly := !commitAuthorPresent && repositoryAuthorPresent
+	commitAndRepositoryAuthorMatch := commitAuthorPresent && repositoryAuthorPresent && rc.Commit.Author.GetLogin() == rc.Author.GetLogin()
```

Comment:
Not sure if we can rely on `rc.Commit.Author.GetLogin` in this case. Looking at the code of the github lib, there is:

```
	// The following fields are only populated by Webhook events.
	Login *string `json:"username,omitempty"` // Renamed for go-github consistency.
```

I think login will in our case never be set. 

Best to run some example requests using for example `curl` to look at the actual JSON returned.

Reply:
@hferentschik yes, you are right login at the commit author level is unreliable. I've switched to use email.